Backpack/Badge/Classes/Badge.swift

```diff
@@ -0,0 +1,109 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objc
+public class BPKBadge: UIView {
+    private let label = BPKLabel(fontStyle: .textXsEmphasized)
+    
+    public var message: String? {
+        get { label.text }
+        set { label.text = newValue }
+    }
+    
+    public var type: BPKBadgeType = .success {
+        didSet { updateLookAndFeel() }
+    }
+    
+    public convenience init(type: BPKBadgeType = .success, message: String) {
+        self.init(frame: .zero)
```

Comment:
The init with frame already calls setup() is it still needed to call `setup()` at the end of this init?

Reply:
good catch!

Reply:
I replaced calling setup with just calling updateLookAndFeel, since didSet are not called if inside an init and didn't want to add a `defer` statement